app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()  # Завантажуємо змінні з .env
+
+API_KEY = os.getenv("API_KEY")
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Kyiv"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {
+        "key": API_KEY,
+        "q": FILTERING
+    }
+    response = requests.get(URL, params=params)
+
+    print(response.json())
```

Comment:
Consider adding error handling for the API request. You can check if the response status code is 200 before attempting to parse the JSON. This will help handle cases where the API request fails or returns an error.